test/functional/services/index.ts

```diff
@@ -47,7 +47,12 @@ import { RemoteProvider } from './remote';
 import { RenderableProvider } from './renderable';
 import { TableProvider } from './table';
 import { ToastsProvider } from './toasts';
-import { PieChartProvider, ElasticChartProvider } from './visualizations';
+import { DataGridProvider } from './dagta_grid';
```

Comment:
I think there is a typo in the filename 😄

Reply:
thank you